### ISSUE TITLE
Accessing the "head title" element

### ISSUE THREAD

**Issue by rickclare** [keathley/wallaby#53]
I'm trying to access the `<title>` element in a HTML document, so that I can assert its contents, but I'm not having any luck.  Is this possible with Wallaby at the moment?

I've tried the following:

Given the following html snippet from a "home-page":

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>My Title</title>
  </head>
  <body>
    This is the home-page
  </body>
</html>
```

And the following tests:

```elixir
defmodule MyApp.HomeFeatureTest do
  use MyApp.FeatureCase

  test "home-page body is present", %{session: session} do
    text =
      session
      |> visit("/")
      |> find("body")
      |> text

    assert text == "This is the home-page"
  end

  test "home-page head title is present", %{session: session} do
    text =
      session
      |> visit("/")
      |> find("title", visible: false)
      |> text

    assert text == "My Title"
  end

  test "home-page head title is present, part 2", %{session: session} do
    text =
      session
      |> visit("/")
      |> find("head title", visible: false)
      |> text

    assert text == "My Title"
  end
end
```

2 out of 3 tests fail:

```
$ mix test test/features/homepage_feature_test.exs                                                           

  1) test home-page head title is present (MyApp.HomeFeatureTest)
     test/features/homepage_feature_test.exs:27
     Assertion with == failed
     code: text == "My Title"
     lhs:  ""
     rhs:  "My Title"
     stacktrace:
       test/features/homepage_feature_test.exs:34



  2) test home-page head title is present, part 2 (MyApp.HomeFeatureTest)
     test/features/homepage_feature_test.exs:37
     Assertion with == failed
     code: text == "My Title"
     lhs:  ""
     rhs:  "My Title"
     stacktrace:
       test/features/homepage_feature_test.exs:44



Finished in 1.1 seconds (0.1s on load, 0.9s on tests)
3 tests, 2 failures
```

**Comment by keathley** [comment]
@rickclare Thanks for submitting this.

The issue here is that `text` only returns what webdriver believes to be "rendered text" (meaning text that is visible to the user). Since the title isn't considered rendered text it returns an empty string. But, we can get the title directly from webdriver. I have a PR to add this here: https://github.com/keathley/wallaby/pull/54. I just need to clean it up and it will be good to merge and release.

**Comment by rickclare** [comment]
That's great @keathley, thanks for super quick response. The rest of Wallaby is working really brilliantly for us, keep up the good work!

Let me know if you need help testing PR#54

**Comment by rickclare** [comment]
@keathley Thanks for sorting #54 

I've just tried out the master branch and my tests are now working.  I'm happy for you to close this issue, unless you've got anything more to add?

Thanks again.

**Comment by keathley** [comment]
This has been released with 0.6.0. Lemme know if you have other issues.
